Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -7769,8 +7771,8 @@ static bool isGridTrackFixedSized(const CSSValue& value)
     auto& function = downcast<CSSFunctionValue>(value);
     if (function.name() == CSSValueFitContent || function.length() < 2)
         return false;
-    return isGridTrackFixedSized(downcast<CSSPrimitiveValue>(*function.item(0)))
-        || isGridTrackFixedSized(downcast<CSSPrimitiveValue>(*function.item(1)));
+    return isGridTrackFixedSized(Ref { downcast<CSSPrimitiveValue>(*function.item(0)) })
```

Comment:
Same as above. If callers have to do `Ref` then maybe we should just change the argument type.